code/modules/events/ion_storm.dm

```diff
@@ -502,61 +504,131 @@
 	var/random_player = "The Captain"
 	if(players.len)
 		random_player = pick(players)		//Random player's name, to be used in laws.
+
+	var/random_department = "Science"
+	var/list/departments = list(	"Science",
```

Comment:
```suggestion
	var/static/list/departments = list("Science",
```
please fix the indentation below

this should be statically typed, it will never be modified

Reply:
Fixed the indentation and made the two lists static.